### ISSUE TITLE
Non-square training

### ISSUE THREAD

**Issue by videscar** [lyuwenyu/RT-DETR#540]
Hello! 
First of all, thanks for you work on the RT-DETR and RT-DETRv2, looks really good.

I have a question, I want to fine-tune the model on my custom dataset, but my images are 1920x768 (wxh), which doesn't much the pretrained weights expected input. I want to avoid resizing my images so I don't loss details.

Is there a way to still use the pretrained weights on my non-square custom resolution? Or the only option I have is training the model from scratch?

Thank you very much for the support!
